### PR TITLE
deletes junk files from given folders

### PR DESCRIPTION
**Pull request by srihithamandalapu**

### CODE REVIEW

**Comment by ANIRUDH-GHUB** [comment]
fixes #9

File: deleteJunkFiles.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                
+        fileList = os.listdir(filePath)
+        folderSizeAfterDeletion = len(fileList)
+        
+        print("Successfully deleted ",folderSizeBeforeDeletion-folderSizeAfterDeletion ,"files from ",filePath)
+        print("Failed to delete ",undeletedFiles,"file/files")         
+    
+#Example call showing temp file deletion by passing some junk folder paths
+ 
+junkFilePaths = ["C://Windows//Prefetch","C://Windows//Temp","C://Users//WELCOME//AppData//Local//Temp"]
```
Comment (ANIRUDH-GHUB) [inline deleteJunkFiles.py:25]: Please add this list to constants

Comment (srihithamandalapu) [reply deleteJunkFiles.py:25]: yeah sure

Comment (ANIRUDH-GHUB) [inline deleteJunkFiles.py:25]: Pleae add this  list to constansta

**Review by ANIRUDH-GHUB — CHANGES_REQUESTED** [review]

**Review by ANIRUDH-GHUB — APPROVED** [review]

**Pull request merged**